File: app/src/main/java/yifimovies/tittojose/me/yifi/homescreen/movieslist/MoviesListView.java
package yifimovies.tittojose.me.yifi.homescreen.movieslist;

import java.util.List;

import yifimovies.tittojose.me.yifi.api.model.Movie;

/**
 * Created by titto.jose on 28-02-2018.
 */

public interface MoviesListView {

    void displayMoviesList(List<Movie> movies);

    void displayErrorWhenNoMoviesLoaded(String errorMessage);
}

File: app/src/main/java/yifimovies/tittojose/me/yifi/homescreen/movieslist/MoviesListView.kt
package yifimovies.tittojose.me.yifi.homescreen.movieslist

import yifimovies.tittojose.me.yifi.api.model.Movie

/**
 * Created by titto.jose on 05-03-2018.
 */
interface MoviesListView {

    fun displayMoviesList(movies: List<Movie>)

    fun displayErrorWhenNoMoviesLoaded(errorMessage: String)
}